### PR TITLE
Nodejs version is set to 10.4, parametric organization name is added

### PR DESCRIPTION
**Pull request by alisengul53**
For all compatibility tests(K8s and backward), node version was `14` it is set to `10.4` now
Organization name is set as parametric.
When a developer would like to run compatibility test with his forked repo, he can set the `organiation_name` parameter to his github name.

### CODE REVIEW

**Review by srknzl — COMMENTED** [review]
I am not sure if kubernetes tests' version should be 10.4, too. WDYT? Also should we add the organization input to other workflows as well?

**Comment by alisengul53** [comment]
We are not changing the K8s' node version, we are chaning the server_compatibily and client_compatibility node version to 10.4

> Also should we add the organization input to other workflows as well?

Make sense, let me also add it to all workflows

**Comment by srknzl** [comment]
> We are not changing the K8s' node version, we are chaning the server_compatibily and client_compatibility node version to 10.4

 Yeah I know. I am not sure we should change for them too. Should we?

**Comment by alisengul53** [comment]
Actually it would be good to have a common supported version. But let me try it first because for internal K8s dockerfile is using node:14 base image and I need to be sure there will be no problem during image creation

File: .github/workflows/server_compatibility.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ on:
       hz_version:
```
Comment (srknzl) [inline .github/workflows/server_compatibility.yaml:10]: I think only this file's organization name is left. By the way, we can omit adding to some files if it is not reasonable to add for some files.

**Review by srknzl — APPROVED** [review]
looking good with a comment

File: .github/workflows/server_compatibility.yaml (hunk 1 above)
Comment (alisengul53) [reply .github/workflows/server_compatibility.yaml:10]: For server compatibility workflow, directly released client versions are used. I think we don't need organization_name parameter for this

**Pull request merged**